transport/src/main/java/io/netty/channel/AdaptiveRecvByteBufAllocator.java

```diff
@@ -132,7 +136,7 @@ public void readComplete() {
      * parameters, the expected buffer size starts from {@code 1024}, does not
      * go down below {@code 64}, and does not go up above {@code 65536}.
      */
-    private AdaptiveRecvByteBufAllocator() {
+    AdaptiveRecvByteBufAllocator() {
```

Comment:
Why not public? It seems this would be the replacement for `DEFAULT`, which was public.